__tests__/utils.test.js

```diff
@@ -28,21 +27,29 @@ describe("Northcoders News API", () => {
     });
 
     describe("/api/topics/", () => {
-      test("responds with an array of topic objects", () => {
+      test("200 - responds with an array of topic objects", () => {
         return request(app)
           .get("/api/topics")
           .expect(200)
           .then((response) => {
-            return response.body;
-          })
-          .then((data) => {
-            const expectedOutput = data.map((elem) => {
+            const expectedOutput = response.body.map((elem) => {
               expect(elem.hasOwnProperty("slug")).toBe(true);
               expect(elem.hasOwnProperty("description")).toBe(true);
             });
```

Comment:
Few points on this task 1 work:

1. You should be sending data back on a key that describes the data e.g. `topics` instead of directly on the response.body

2. If you use a loop to assert each topic in the array has the right structure you open yourself to false positives. For example, if the response array is empty the map won't run and no assertions will fail so we'll get a passing test. To counter this we want to always make sure the array has something inside it (or the right num of items inside it) alongside any loops we use to do further assertions